data/json/items/containers.json

```
@@ -3328,5 +3328,55 @@
     ],
     "qualities": [ [ "CONTAIN", 1 ] ],
     "flags": [ "COLLAPSE_CONTENTS" ]
+  },
+  {
+    "id": "paint_can_steel",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "metal paint can" },
+    "description": "A metal paint can holding a gallon of liquid.",
```

```suggestion
    "description": "A metal paint can, able to hold a gallon of liquid.",
```
Can't have descriptions lying to you if the can is empty 😛

---

Good catch, I have a bad habit of using thought shortcuts in text. I'll fix it right away.